### ISSUE TITLE
coala on MacOS

### ISSUE THREAD

**Issue by anubhabsen** [coala/coala#4177]
### Overview Description
Not able to execute coala from terminal after installation in MacOS

# Steps to Reproduce

1. Use MacOS
2. pip3 install coala-bears
3. try executing coala/ coala-bears

# Actual Results

zsh: command not found: coala

# Expected Results

analysis of code/ usage instructions

**Comment by ramonh** [comment]
(this is sils on another persons PC :)) CC @Udayan12167

**Comment by Makman2** [comment]
Did the installation succeed? Do you have the install log?

**Comment by anubhabsen** [comment]
Not via pip, but I did succeed in installing it with the setup file in the repo. Encountered some issues with locales too. So commented out line 17-23 of setup.py and that worked.

**Comment by Makman2** [comment]
What issues?
What was your install command for the `setup.py` file?
And are you sure that your path is correct?

**Comment by anubhabsen** [comment]
`python3 setup.py`
Executing before commenting out gave an locale error. Tried exporting all locales to en-us and c, neither solved it, tried to turn off Set locale environment variables on startup in preferences, that didn't work either.

**Comment by Makman2** [comment]
Could you paste the exact error?

**Comment by anubhabsen** [comment]
```
Traceback (most recent call last):
  File "setup.py", line 21, in <module>
    locale.setlocale(locale.LC_ALL, 'C.UTF-8')
  File "/usr/local/Cellar/python/2.7.13/Frameworks/Python.framework/Versions/2.7/lib/python2.7/locale.py", line 581, in setlocale
    return _setlocale(category, locale)
locale.Error: unsupported locale setting
```

**Comment by Makman2** [comment]
Why does the path there point to a Python 2.7 installation?

**Comment by anubhabsen** [comment]
Oops, `python3 setup.py` did the job. But running coala now gives
`[ERROR][22:21:12] There is a conflict in the version of a dependency you have installed and the requirements of coala. This may be resolved by creating a separate virtual environment for coala or running `pip install "coala~=0.10.0"`. Be aware that the latter solution might break other python packages that depend on the currently installed version.`
And `pip install coala` gives 
`Command "python setup.py egg_info" failed with error code 4 in /private/var/folders/hq/jp648pdd6jl9353x43jr70rw0000gn/T/pip-build-iORSrV/coala/
`
pip3 install coala installs it, but running it immediately after gives command not found (not anymore since I already installed with the setup.

**Comment by SanketDG** [comment]
@anubhabsen please ask in gitter chat, more people will be able to help you there 😄

**Comment by Makman2** [comment]
1. Please use `pip3` instead of `pip` as coala only works with Python 3.
2. So you can execute coala now?

Btw could you be a bit more precise? It's a bit hard to follow you^^

**Comment by anubhabsen** [comment]
Using `pip3` to install does install coala (it shows up at `pip3 list`), but running coala gives command not found.
Now I can execute it since I installed it manually using the setup.py file after cloning the repo.

**Comment by Makman2** [comment]
Ah okay :) So problem solved? :)

**Comment by anubhabsen** [comment]
Yes! Thank you!
